### PR TITLE
Use modern urlpatterns syntax

### PR DESCRIPTION
**Pull request by noisecapella**

### CODE REVIEW

File: server_status/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=missing-docstring
-__version__ = '0.3.2'  # pragma: no cover
```
Comment (bdero) [inline server_status/__init__.py]: Can we wait to bump the version until after the release-candidate branch is already cut to align with our release process? https://github.com/mitodl/release-script/blob/master/README.md

**Comment by pdpinch** [comment]
@noisecapella @bdero what needs to be done so we can merge this PR?

**Comment by noisecapella** [comment]
Nothing on my end. @bdero what do you think?

**Comment by bdero** [comment]
I'm fine with proceeding, I had a comment that requires action to address but it's more of a nit than anything. :+1:

**Comment by pdpinch** [comment]
Sorry, I honestly didn't understand your comment. 

Are you saying we should revert the version bump from this PR and then apply it as part of the release process?

Or is it something else?

**Comment by bdero** [comment]
Yes, that's what I meant in the comment.

**Comment by noisecapella** [comment]
Ok, I removed the version change

**Pull request merged**